test/space_calculation_test.rb

```diff
@@ -112,6 +112,8 @@ def expect_to_execute(command)
     end
   end
 
+# storage-ng
+=begin
```

Comment:
NP: Um, personally I do not like these block comments, it breaks simple grepping the sources when you are looking for something, it might confuse some simple syntax highlighters and you can easily overlook that the block is commented when the block it too long.

I'd prefer the usual '#' comments, editors/IDEs usually have some shortcuts for (un)commenting multiple lines.